mlir/utils/performance/perfRunner.py

```diff
@@ -1285,7 +1308,7 @@ def getNumCU(chip):
         rocminfo = subprocess.check_output("/opt/rocm/bin/rocminfo",
                                            stderr=subprocess.PIPE)
     except subprocess.CalledProcessError as e:
-        print(e.stderr.decode('utf-8'))
+        print(f"Process error: {e.stderr.decode('utf-8')}")
```

Comment:
Still trying to identify the cause of the intermittent rocminfo failures.  Highlight this case in the log file.